pkg/commands/drift/drift.go

```diff
@@ -79,21 +93,51 @@ func Process(ctx context.Context, organizationID, bucketQuery, driftignoreFile s
 		return nil, fmt.Errorf("failed to expand graph for ignored assets: %w", err)
 	}
 
-	gcpIAM, err := actualGCPIAM(ctx, organizationID, foldersByID, projectsByID)
+	g, gCtx := errgroup.WithContext(ctx)
```

Comment:
I don't know how this will be used in practice, but should we be worried about issuing too many concurrent API requests and getting throttled? Perhaps use https://pkg.go.dev/golang.org/x/sync/errgroup#Group.SetLimit.

Reply:
Yeah. Good idea. Was waiting to get feedback on errGroup vs WaitGroup vs something else like https://github.com/sourcegraph/conc

Reply:
I'd love to start using conc, maybe this is a good time. Though I imagine @sethvargo will have strong feelings here and he's OOO. If you want to take a stab at using conc, I'd approve it.

The safe choice in my opinion would be to stick with errgroup for now. I'd choose it over WaitGroup because it saves you from managing errors yourself.

Reply:
Using conc: https://github.com/abcxyz/guardian/pull/118/files

Reply:
Dropped a comment on that (#118) suggesting a simplification. The conc approach seems good to me.

Reply:
We have a thing that does this and is safer (both type and error safe): https://github.com/abcxyz/pkg/tree/main/worker